build-tools/packages/build-cli/src/library/package.ts

```diff
@@ -149,7 +150,7 @@ export async function npmCheckUpdates(
 		log?.verbose(`Checking packages in ${path.join(repoPath, glob)}`);
 
 		// eslint-disable-next-line no-await-in-loop
-		const result = (await ncu({
+		const result = (await ncu.run({
```

Comment:
![image](https://github.com/microsoft/FluidFramework/assets/716334/7dd49a98-f974-4725-9f55-b40f8347de3b)
"Why are we using... oh, never mind" 😆